Capstone branch

abdulrahmanshr75 opened the pull request:
Gamer developers conference.
- build two pages of the website about and home.
- added responsive toggle menu bar for the mobile version.
- added javascript dynamical data renders inside the website.


https://www.loom.com/share/c828b938e45043e0bafeb7c941c7e018

abdulrahmanshr75 committed:
new branch

abdulrahmanshr75 committed:
Update Readme

abdulrahmanshr75 committed:
update screenshot

abdulrahmanshr75 committed:
manipulating navbar toggle

abdulrahmanshr75 committed:
update about

abdulrahmanshr75 committed:
fixed html navbar toggle

abdulrahmanshr75 committed:
fixing linters

abdulrahmanshr75 committed:
implemented javascript

abdulrahmanshr75 committed:
add speakers images and descriptions

DammyShittu commented on index.html:
```diff
@@ -0,0 +1,184 @@
+      <img class="Logo" src="images/GDC.png" alt="Logo" />
+      <div class="navbar-links">
+        <ul>
+          <li><a href="about.html">ِِAbout</a></li>
```
- The link to the `About` page is currently not working, both locally and on GitHub pages. I noticed you named your file `About.html` and the reference to the page, `about.html`. Kindly fix this 😉

DammyShittu commented on About.html:
```diff
@@ -0,0 +1,146 @@
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-content">
+        <img class="logo6" src="images/GDC.png" alt="logo" />
+        <h2 class="footer-text">
+          2022 Gamer Developer Conference San Francisco, some rights reserved.
+        </h2>
+      </div>
+    </footer>
```
- Kindly follow the background color and other color guidelines used for the footer of the About page 😀

DammyShittu commented on index.html:
```diff
@@ -0,0 +1,184 @@
+            class="item-img margin0"
+            alt="lecturer"
+          />
+          <h2 class="item-title margin0">Lecture</h2>
```
- Headings are meant to have a bigger font size with respect to the text around them. Kindly increase the font size of the Main Program Headings(Lecture, Exhibition, Forum, Workshop, and Ignite). This will improve user experience as it would allow users to see the main things/activities that would be done 👍

DammyShittu commented on index.html:
```diff
@@ -0,0 +1,184 @@
+      <a class="program" href="#">SEE THE WHOLE PROGRAM</a>
+      <a class="btn" href="#">Join GDC Global summit 2022</a>
+    </section>
+    <section class="third-section">
+      <h2 class="text5">Featured Speakers</h2>
+      <div class="line2"></div>
+      <div class="speakers"></div>
+    </section>
```
- For the desktop view, what do you think about displaying the featured speakers' section in two columns, and three rows just like the template shows?

- Your Design 👇 

![](https://i.imgur.com/yVZYlQq.png)

- Original Design 👇

![](https://i.imgur.com/IIIov7h.png)

DammyShittu commented on index.js:
```diff
@@ -0,0 +1,65 @@
+
+const cards = document.querySelector('.speakers');
+for (let i = 0; i < data.length; i += 1) {
+  const Card = document.createElement('div');
```
- I would suggest you begin your variable names with lowercase letters, as according to JavaScript naming convention, class constructors are normally instantiated with uppercase letters at the beginning. I hope this makes sense 😁 

You can also go through this [discussion](https://teamtreehouse.com/community/does-the-first-letter-of-a-variable-name-have-to-be-lowercase) for more information

DammyShittu commented on index.html:
```diff
@@ -0,0 +1,184 @@
+      <div class="line2"></div>
+      <div class="speakers"></div>
+    </section>
+    <section class="partner">
+      <h2 class="text4">Partner</h2>
+      <div class="line2"></div>
+      <div class="partner-flex">
+        <img class="partner-icon" src="images/mozella.png" alt="mozella-icon" />
+        <img class="partner-icon" src="images/google.png" alt="google-icon" />
+        <img class="partner-icon" src="images/airbnb.png" alt="airbnb-icon" />
+      </div>
+    </section>
```
- Your Partners' section looks amazing. 💯 However, I think it'll look more awesome when you add the other partners just as the template shows. 😉

- Your Design 👇 

![](https://i.imgur.com/OG6rPkR.png)

- Original Design 👇 

![](https://i.imgur.com/EHX5KAf.png)

DammyShittu reviewed (changes requested):
Hi @abdulrahmanshr75 , 

Amazing work so far. Your project looks absolutely fantastic 🚀

However, there are some issues that you still need to work on to get this project approved, but you are ALMOST THERE!


# REQUIRED CHANGES ♻️

_Check the comments under the review._

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Changes Requested

- the About page is currently not working renamed it to about.html
- followed the background color and other color guidelines used for the footer of the About page.
-  increase the font size of the Main Program Headings.
-  displaying the featured speakers' section in two columns, and three rows.
- begin my variable names with lowercase letters.
- add the other partners.

moise10r commented on style.css:
```diff
@@ -0,0 +1,693 @@
+@font-face {
+  font-family: goose;
+  src: url(fonts/Cocogoose_trial.otf);
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+}
+
```
- [ ] I can see that your website present some overflow as you can see bellow :point_down: 

![222](https://user-images.githubusercontent.com/57562869/148445379-6d06e4a2-3e44-43f2-969f-de1691bc88d3.png)

- I think that can be solved but doing : 

```
* {
  overflow: hidden;
}

html {
  overflow: scroll;
  width: 100%;
}
```

moise10r commented on index.html:
```diff
@@ -0,0 +1,190 @@
+      <img class="Logo" src="images/GDC.png" alt="Logo" />
+      <div class="navbar-links">
+        <ul>
+          <li><a href="about.html">ِِAbout</a></li>
```
- [ ]  As required by the first reviewer. The link to the About page is currently still not working, both locally and on GitHub pages.  Kindly fix this issue and make sure it works :factory_worker:

moise10r commented on style.css:
```diff
@@ -0,0 +1,693 @@
+  width: 60px;
+}
+
+.flexbox {
+  margin-top: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
```
- [ ] I can see that you have an issue with the width of the Main Program section container :point_down: 

![111](https://user-images.githubusercontent.com/57562869/148447650-5272db17-635d-4d53-bd4f-f404fb1cf0f5.png)

- I think you miss a `width: ` property, kindly add width to this container to solve the issue eg (`width: 90%`)

moise10r reviewed (changes requested):
### **Hi @abdulrahmanshr75 **  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### <FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Changes requested 2

- the about file in my local drive is about not About and it is working fine.
- adjusted the width of the Main Program section container.
- fixed the overflow problem.

abdulrahmanshr75 committed:
rename about.html

abdulrahmanshr75 committed:
fixed the overflow

abdulrahmanshr75 committed:
Merge branch 'capstone-branch' of https://github.com/abdulrahmanshr75/capstone-1 into capstone-branch

abdulrahmanshr75 committed:
fix the styles

abdulrahmanshr75 committed:
update readme

abdulrahmanshr75 committed:
update styles

abdulrahmanshr75 committed:
update styles

abdulrahmanshr75 committed:
update styles

moise10r reviewed (approved):
###  **Hi @abdulrahmanshr75 ,** 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Cheers and Happy coding!👏👏👏


 #### Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
